### ISSUE TITLE
[ Enhacement ] Better UI for About Us Page

### ISSUE THREAD

**Issue by Kajol-Kumari** [HITK-TECH-Community/Community-Website#308]
Deliverables:

- [ ] Better UI for about Us page.

**Note** - We will be accepting multiple PRs for this but Please make sure to show your design here and only after we give you confirmation, come up with a PR.

**Comment by kenkirito** [comment]
assign me

**Comment by ashuvssut** [comment]
Hello mam, I wish to work on improving the team section.
![image](https://user-images.githubusercontent.com/60546840/106329283-0796a000-6236-11eb-98e7-820e0ac6a4d8.png)
I saw the code in about_us.js and found that every thing(content in the cards) is hardcoded.
Do you have any plans to generate all this cards by some json from the backend?

----------------

in https://swoc.tech/project.html there is a project section that uses the swiper.js npm package. I can make that with multiple rows. I think 2 rows will be good looking.

**Comment by Kajol-Kumari** [comment]
Hey @ashuvssut yes, we will be populating the cards using the json data coming from backend. We don't want any slider effects for the teams section. Just good looking card with standard hover effect would work

**Comment by ashuvssut** [comment]
okay! no fancy swiper sliders.

Please tell me how you want the team section cards to be...
- no fixed number of columns, cards adjust themselves automatically (upto 5 coulmns), or
- 4 columns, or
- 3 columns

**Comment by Kajol-Kumari** [comment]
> okay! no fancy swiper sliders.
> 
> Please tell me how you want the team section cards to be...
> 
> * no fixed number of columns, cards adjust themselves automatically (upto 5 coulmns), or
> * 4 columns, or
> * 3 columns

Yeah works We just want two card of `Founder` and `Co-founder` look a bit different from the rest

**Comment by ashuvssut** [comment]
okay i get it.
first row cards will be populated only and only if the key values in json are `Founder` / `Co-founder` otherwise,
others(key value :`member`) will get filled in 2nd row which will be a flexbox with ` flex-wrap: wrap;`. Thus, both rows will look completely different. 

I will start working on it. 
will submit PR within 4days.

**One more question**: should I push my test JSON file also so that you can test it when I submit my PR? If yes, where should i keep that json (some `test-directory/` or something?).

**Comment by Kajol-Kumari** [comment]
> okay i get it.
> first row cards will be populated only and only if the `role` key's values in json is `Founder` / `Co-founder` otherwise,
> others(`role` key's value :`member`) will get filled in 2nd row which will be a flexbox with ` flex-wrap: wrap;`. Thus, both rows will look completely different.
> 
> ----------Row1------------
>              |    | |     |
> ----------Row1------------
> ----------Row2------------
> |   | |    | |    | |     | |    | |    |
> |   | |    | |    | |     | |    | |    |
> |   | |    | |    | |     | |    | |    |
> |   | |    | |    | |     | |    | |    |
> |   | |    | 
> ----------Row2------------
> I will start working on it.
> will submit PR within 4days.
> 
> **One more question**: should I push my test JSON file also so that you can test it when I submit my PR? If yes, where should i keep that json (some `test-directory/` or something?).

Hey! You can make a folder named as `test_directory` and put the data in a file there only.

**Comment by ashuvssut** [comment]
okay! thx

**Comment by jatinjain001** [comment]
I would like to work on this , If it is available please assign it to me

**Comment by ashuvssut** [comment]
@Kajol-Kumari please assign me.

I have been working on this 😓

**Comment by Kajol-Kumari** [comment]
Hey @ashuvssut and @jatinjain001 feel free to share the new UI design that you come with here. We will be accepting multiple PRs for this issue :)

**Comment by ashuvssut** [comment]
Ah relief 🙃

**Comment by jatinjain001** [comment]
okay

**Comment by Kajol-Kumari** [comment]
Just a check, are you guys working on it? 👀

**Comment by ashuvssut** [comment]
Just 2 days. I ll do it. I forgot about this one (because it was not showing in my GH account's   'assigned issues' tab)

**Comment by Kajol-Kumari** [comment]
> Just 2 days. I ll do it. I forgot about this one (because it was not showing in my GH account's 'assigned issues' tab)

Thanks for the update 😄
Looking forward to your PR 😁

**Comment by sahildotexe** [comment]
Hey !! I 'm a participant of GSSOC'21, can you assign this issue to me?

**Comment by jackfrost13** [comment]
@Kajol-Kumari Is this still open? I see a merged PR

**Comment by Shelly011s** [comment]
Is this still open for DWOC?
